package.json

```diff
@@ -27,7 +27,7 @@
     "netmask": "^1.0.5",
     "promise": "^7.0.4",
     "pulse-publisher": "^1.3.1",
-    "remotely-signed-s3": "^2.0.0",
+    "remotely-signed-s3": "^3.0.0",
```

Comment:
This was r+'ed in another PR

Reply:
yep, but not merged in yet, so it showed up here as well